### PR TITLE
Rename OPENSHIFT_OFFLINE_TOKEN to MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN

### PR DESCRIPTION
**Pull request by lucamolteni**
NO-JIRA

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [x] Check if code and Github action workflows have been modified and if the modification is safe
- [x] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

File: dev/bin/manager-run.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # Env vars:
 # - MANAGED_CONNECTORS_NAMESPACE_ID: namespace where managed connectors will be deployed (required only if MC actions are used, default="empty")
 # - MANAGED_CONNECTORS_CONTROL_PLANE_URL: endpoint of the MC Control plane. (required only if MC actions are used, default="empty")
-# - OPENSHIFT_OFFLINE_TOKEN: Red Hat account offline token (required, get it at https://console.redhat.com/openshift/token)
+# - MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN: Red Hat account offline token used by the fleet manager to authenticate to manage connectors (required)
```
Comment (manstis) [inline dev/bin/manager-run.sh:9]: If I wanted to be really pedantic I'd suggest this should be "... to manage**d** connectors..."

**Review by manstis — APPROVED** [review]

**Review by robpblake — APPROVED** [review]
@lucamolteni Looks fine to me. I know it's a pain, but can you please create a JIRA for this otherwise we loose visibility on these small tasks that take up our time.

Can you also please update the environment spreadsheet to match the rename as well please to avoid any more confusion.

cheers,

Rob

**Review by r00ta — CHANGES_REQUESTED** [review]
I think you missed to update `dev/README.md` and `startMinikubeDeployLocalDev.sh`

**Comment by lucamolteni** [comment]
I forgot this one as well  (has to be done when it's merged)
https://docs.google.com/spreadsheets/d/1Ud8BnqJJs6O0nZtD8o4WIFTqExcrfvofjCqWqJtX8YI/edit#gid=0

**Comment by lucamolteni** [comment]
> @lucamolteni Looks fine to me. I know it's a pain, but can you please create a JIRA for this otherwise we loose visibility on these small tasks that take up our time.

https://issues.redhat.com/browse/MGDOBR-672

**Review by sutaakar — APPROVED** [review]

**Review by r00ta — APPROVED** [review]

**Pull request merged**